mm2src/coins/lightning/ln_utils.rs

```diff
@@ -215,15 +214,14 @@ pub async fn init_channel_manager(
     };
 
     // Update best block whenever there's a new chain tip or a block has been newly disconnected
-    spawn(ln_best_block_update_loop(
-        platform,
+    platform.spawner().spawn(ln_best_block_update_loop(
```

Comment:
It seems that we need to use a separate `spawner` for LightningCoin and maybe other protocols if they also rely on platform spawner:
When LightningCoin is disabled and platform coin is not, lightning-related futures won't be aborted.
I guess there is also a chance to call `abort_call` from lightning context, which will unexpectedly abort platform coin related futures.
cc @shamardy

Reply:
`platform` here is `UtxoStandardCoin` or `UtxoArc`. It's [lightning::ln_platform::Platform](https://github.com/KomodoPlatform/atomicDEX-API/blob/stop-spawned-futures/mm2src/coins/lightning/ln_platform.rs#L205).
When the lightning coin is enabled, we create a subsystem from the `UtxoStandardCoin`:
https://github.com/KomodoPlatform/atomicDEX-API/blob/c904c63e3036646c9208e9b411b406d6121248b0/mm2src/coins/lightning/ln_platform.rs#L174-L183

Reply:
Oh, I have not noticed this, thanks for the info!